common/buildings/00_duchy_capital_buildings.txt

```diff
@@ -341,6 +341,9 @@ siege_works_02 = {
 	character_modifier = {
```

Comment:
Localization of `warfare_almost_tamed_wild_horse_modifier` should be adapted to new modifiers.

Reply:
Why you marked it resolved?